cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -123,6 +103,26 @@
 #   The st library for Low Bandwidth Plugins if found (rtilbst).
 # - ``RTIConnextDDS::rtizrtps``
 #   The rtizrtps library if found (rtizrtps).
+# - ``RTIConnextDDS::routing_service_infrastructure``
+#   The infrastructure library for Routing Service if found.
+# - ``RTIConnextDDS::routing_service_c``
+#   The C API library for Routing Service if found (includes
+#   rtiroutingservice, rtirsinfrastructure, rtixml2 and rticonnextmsgc. Also
```

Comment:
`rtiroutingservice, rtirsinfrastructure, rtixml2 and rticonnextmsgc; also`

(I'm being really picky here, you don't have to do this if you don't want ...)